### ISSUE TITLE
Unnecessary warning message in the log.

### ISSUE THREAD

**Issue by huebl** [ASNeG/OpcUaStack#226]
**Describe the bug**
There is an unnecessary warning message in the log.

```
2019-May-19 10:04:38.613202 DBG [7f725fee6700] change state in session service : SessionServiceName=<>, OldState=<None>, NewSate=<Disconnected>
2019-May-19 10:04:38.613267 WRN [7f725fee6700] update callback not available
2019-May-19 10:04:38.613394 DBG [7f725ccce700] async connect event : SessId=<7>
```

**Expected behavior**
The log message should be revised.

**Comment by huebl** [comment]
The following log messages should also be checked.

```
2019-May-19 10:04:38.615555 INF [7f725ccce700] receive acknowledge : ChannelId=<6-0>, Address=<127.0.0.1>, Port=<8889>
2019-May-19 10:04:38.615701 ERR [7f725ccce700] certificate vector empty
2019-May-19 10:04:38.615915 DBG [7f725ccce700] handle write complete - OpenSecureChannelRequest : ChannelId=<6-0>
2019-May-19 10:04:38.616776 ERR [7f725ccce700] input parameter error
```

**Comment by huebl** [comment]
The following log messages should also be checked.

```
2019-May-19 10:04:38.619975 DBG [7f725ccce700] change state in session service : SessionServiceName=<WebGateway-1>, OldState=<GetEndpoint>, NewSate=<Reconnecting>
2019-May-19 10:04:38.620131 ERR [7f725ccce700] opc ua secure channel read message header error; close channel : ChannelId=<6-3>, Message=<Bad file descriptor>
2019-May-19 10:04:38.620312 ERR [7f725ccce700] opc ua secure channel close : Id=<3>, AsyncSend=<0>, CloseFlag=<0>
2019-May-19 10:04:38.620421 INF [7f725ccce700] secure channel to server closed : ChannelId=<6-3>, Address=<127.0.0.1>, Port=<8889>
2019-May-19 10:04:38.620523 DBG [7f725ccce700] handle disconnect event : SessId=<7>
```
